umap/static/umap/js/umap.features.js

```diff
@@ -483,12 +483,30 @@ L.U.FeatureMixin = {
     return false
   },
 
+  parseDateField: function (value) {
+    if (parseFloat(value).toString() === value.toString()) {
+      value = parseFloat(value);
+      if (value < 10000000000) {
```

Comment:
We need at least comments to explain these two values `10000000000` and `1000`.

Reply:
See this comment: https://github.com/umap-project/umap/pull/1491#discussion_r1439922296

Not happy with this part of the implementation, happy to receive more feedback/suggestions from you.